Dockerfile

```diff
@@ -4,7 +4,7 @@ ARG BASE_IMAGE_MINIMAL
 # Build node feature discovery
 FROM golang:1.17.2-buster as builder
 
-# Download the grpc_health_probe bin
+# Download the grpc-health-probe bin
```

Comment:
since we are changing this file, maybe update that it's no longer "Download" but "Build and install"

Reply:
it was Download until #707

Reply:
@mythi thanks. I did it in the new PR: https://github.com/kubernetes-sigs/node-feature-discovery/pull/709